README.md

```diff
@@ -33,142 +33,184 @@ state of your service, interacting with the fundamental APIs:
-          performance_type  = "normal"
-          cpu_core_count    = 2
-          memory_size       = 4
-      }
-      data "huaweicloud_vpc_subnet" "mynet" {
-          name = "xpanse_subnet1"
-      }
-      resource "huaweicloud_compute_instance" "basic" {
-          name               = "basic"
-          image_id           = "a8601887-81d5-4eed-9338-382cf5b6d80b"
-          flavor_id          = data.huaweicloud_compute_flavors.myflavor.ids[0]
-          availability_zone  = data.huaweicloud_availability_zones.myaz.names[0]
-          network {
-              uuid = data.huaweicloud_vpc_subnet.mynet.id
-          }
-      }
+  # kind, Supported values are terraform, pulumi, crossplane.
+  kind: terraform
+  # Context for deployment: the context including some kind of parameters for the deployment, 
+  ## such as env,variable.
+  # - env: The value of the fix parameters are defined by the ISV with the @value at the 
+  ## initial time.
+  # - variable: The value of the variable parameters are defined by the user on the console.
+  # The parameters will be used to generate the API of the managed service.
+  context:
+    - name: HW_REGION_NAME
+      description: huawei cloud region name.
+      kind: env
+      type: string
+      mandatory: true
+      validator: minLength=6|maxLength=26
+    - name: HW_ACCESS_KEY
+      description: Huawei cloud access key.
+      kind: env
+      type: string
+      mandatory: true
+    - name: HW_SECRET_KEY
+      description: Huawei cloud secret key.
+      kind: env
+      type: string
+      mandatory: true
+    - name: admin_passwd
+      description: The admin password of all nodes in the K8S cluster.
+      kind: variable
+      type: string
+      mandatory: true
+      validator: minLength=8|maxLength=16|pattern=^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9]).{8,16}$
+    - name: network_id
+      description: The network id of all nodes in the K8S cluster.
+      kind: variable
+      type: string
+      mandatory: false
+    - name: security_group_id
+      description: The security group id of all nodes in the K8S cluster.
+      kind: variable
+      type: string
+      mandatory: false
+  deployer: |
+    data "huaweicloud_availability_zones" "osc-az" {}
+
+    variable "admin_passwd" {
+    type = string
+    description = "The root password for all nodes in the K8S cluster."
+    }
+
+    variable "flavor_worker_nodes_count" {
+    type = number
+    description = "The number of worker nodes in the K8S cluster."
+    }
+
+    variable "flavor_flavor_id" {
+    type = string
+    description = "The flavor id of all nodes in the K8S cluster."
+    }
+
+    variable "network_id" {
+    type = string
+    default= "6a7c406d-5bc6-4b50-8f75-948aea69cb24"
+    description = "The network id of all nodes in the K8S cluster."
+    }
+
+    variable "security_group_id" {
+    type = string
+    default= "668833c8-459d-4d15-871d-34c3cbcda9d4"
```

Comment:
please try to replace all the ids with `data resources` as below.

Reply:
Ok.